clairmeta/utils/xml.py

```diff
@@ -29,7 +30,7 @@ def prettyprint_xml(xml_str):
     return parsed.toprettyxml(indent="  ")
 
 
-def post_parse_node(path, key, value):
+def post_parse_node(path, key, value, ns_prefix=":"):
```

Comment:
@remia this default is in disagreement with `ns_prefix = "\n"` further down (and it's the unholy colon). I have a feeling that it's intended and with backwards compatibility of public(?) API function `post_parse_node`. Is that what it's about? If so, I think a comment to would help then. What do you think?

Reply:
This was just me don't wanting to update the doctest for that function which still used the ':' character. I'll default to whitespace instead for these tests, using the newline character seems to throw off pytest when looking up the doctest.

Reply:
@remia thanks for the clarification. Why not use space everywhere then? `_DEFAULT_NS_SEP = ' '` and using that at twoplaces would not be good? It would keep things in sync for free :smiley: Also, I only notice now that `ns_prefix` is rather misleading for a name. It's about a namespace separator. Maybe `ns_sep` or `namesep` or or?

Reply:
Great points @hartwork, I think I addressed that in the last commit. To answer an earlier point, `post_parse_node` is not meant to be in public API, granted we are not really rigorous on that front.